### PR TITLE
Minor Fixes

### PR DESCRIPTION
**Pull request by lorenzolewis**
- Adjusts capitalisation for blog title
- Remove Lottie-player dep (no longer used
- Remove themify icons (no longer used)
- Removed dynamic-css plugin (only used for removed themify icons)
- Remove unused `#starfield` attribute from CSS
- Remove used `Icon` and `OSList` from theme modules
- Move crowdin CLI down to dev dep
- Fix index package svgs (extra border)

### CODE REVIEW

File: src/css/custom.css [hunk 1]
```diff
@@ -110,6 +110,10 @@ h2 > [class^='ti-'] {
   min-height: 52px;
 }
 
+[class*='announcementBar_'] a:hover {
+  color: var(--ifm-button-color) !important;
+}
```
Comment (Beanow) [inline src/css/custom.css]: Doesn't look like this resolves as a color for me on Firefox
image
What's the expected result?

Comment (FabianLars) [reply src/css/custom.css]: It changes the link color only on hover

**Comment by FabianLars** [comment]
This makes the announcement links inconsistent with the other broken links 👀 (Quick Start and Learn More)

File: src/css/custom.css (hunk 1 above)
Comment (Beanow) [reply src/css/custom.css]: Right, should have shown with hover emulated.
It's supposed to be the same as the text beside it I assume? So no hover effect essentially.
image

**Comment by lorenzolewis** [comment]
> This makes the announcement links inconsistent with the other broken links 👀 (Quick Start and Learn More)

Lol, love just casually throwing in the other ugly CSS issues here 😂

A few more custom-CSS things than I'd like, but that's what we get when we use blue as a primary brand colour. How's this look for you both now? Updated the original comment with a few more changes I did (general cleanup).

File: src/css/custom.css (hunk 1 above)
Comment (lorenzolewis) [reply src/css/custom.css]: Updated this, lmk what you think

**Comment by FabianLars** [comment]
1) The `Quick Start` buttom still has the "invisible on hover" issue.
2) The announcement link doesn't look like a link without hovering on it (and why would someone move their mouse on it if it doesn't look clickable)

**Comment by lorenzolewis** [comment]
Can you post a screenshot of what you're talking about with #1?

Any ideas on how to help with #2? I tried bolding but it doesn't work

**Comment by FabianLars** [comment]
grafik

**Comment by FabianLars** [comment]
about 2) maybe just re-add the underline and make it disappear on hover? Basically the reverse of all other links.
or add a `here`, i think that could suggest a clickable link too

**Comment by lorenzolewis** [comment]
Give that a try (when the build finishes). The button CSS on the infirma site (backing all the Docusaurus CSS) is different than what they actually ship :)

But maybe that will change once I update to the docusaurus rc because I saw they pulled an update through.

For the announcement bar I just kept the underline all the time because I felt like that makes more sense than inverting the underline stuff, but let me know what you think

**Comment by lorenzolewis** [comment]
Well, now that 2/2 of the extra problems fixed themselves, this was a fun little cleanup PR at the very least, merge ready?

**Review by FabianLars — APPROVED** [review]

**Pull request merged**